templates/default/user/modify_user.sh

```diff
@@ -16,11 +16,23 @@ if [ "$oh" != "$nh" ]
 then
     mkdir -p $(dirname "$nh")
     mv "$oh" "$nh"
-    chown -R {{ user.uidnumber }}:{{ user.gidnumber }} "$nh"
+
+    {% if user.oldgidnumber %}
```

Comment:
this test is I think useless, for user home, because if home moved, it means that his main group has been modified.
but anyway, it only a test which should always be true, so should be fine

Reply:
Yes it is useless check for now, maybe we can keep anyway, as :
- It does not change the current shell script generated 
- I like the idea that everytime when chown is used in the context of user modification, the check is done before run it

Reply:
yeap